src/mipp_impl_AVX.hxx

```diff
@@ -213,7 +240,11 @@
 		return _mm256_castsi256_ps(_mm256_set_epi32(vals[7], vals[6], vals[5], vals[4],
 		                                            vals[3], vals[2], vals[1], vals[0]));
 	}
-
+         template <>
+         inline reg set<uint32_t>(const uint32_t vals[nElReg<uint32_t>()]) {
+            return _mm256_castsi256_ps(_mm256_set_epi32(vals[7], vals[6], vals[5], vals[4],
```

Comment:
As you use `_mm256_set_epi32` (expect `int32_t` type) you should add a casts to `uint32_t`.
`_mm256_set_epi32((uint32_t)vals[7], (uint32_t)vals[6], ...)`